### ISSUE TITLE
Jamtaba do not disconnect from room even though I turn off wifi

### ISSUE THREAD

**Issue by jonjamcam** [elieserdejesus/JamTaba#230]
One day we should work on HAMtaba transmitting via SW, but I think this is a bug.

![jamtaba ghost](https://cloud.githubusercontent.com/assets/15310433/12225295/835e7bc2-b7e6-11e5-82f7-0a4888880bb5.jpg)

First Enter a room normally, then turn off wifi jamtaba keeps connected for about 5-10 seconds

**Comment by elieserdejesus** [comment]
Jon, I tested this issue and see the same behavior here. I debug the code and, after turn off wifi, the network error handlers are invoked only after some seconds. It's not a Jamtaba issue, it's something more deep. It's related with the Operational System and the TCP protocol. I think this is out of our control. The Operational System is waiting some seconds for a kind of "network recover". 
I read some posts about "how detect if a network cable is unplugged". Is possible periodically check the status of the network interface and try detect when the "status" change.
At least Jamtaba is not crashing, it's justing closing and showing a error message after some seconds.

I will keep this issue on the list as a low priority enhancement.

**Comment by jonjamcam** [comment]
Funny. For the record it happens with Reaninjam too (as expected).

**Comment by jonjamcam** [comment]
I'm closing this issue because there's probably no way this can be solved within jamtaba code. It looks more like a OS specific issue.
